pages/desktop/discovery.py

```diff
@@ -35,23 +35,9 @@ class DiscoveryPane(Base):
 
     _featured_addons_base_locator = (By.CSS_SELECTOR, '#featured-addons .addon-title ')
 
-    def __init__(self, testsetup, path):
```

Comment:
It's still there, but it uses a fixture now. Really, these tests just have a different base URL for production, and that's what this is for. I would suggest simply running a separate suite for just these tests and setting the different base URL, except there's one test that uses the application base URL and the services base URL. There's a good chance we can revisit to see if this is the right thing to do, but I wanted as few changes as possible in this particular patch.